docs/content/en/docs/snippets/tasks/install.md

```diff
@@ -11,13 +32,12 @@ kubectl wait --for=condition=Available deployment/klc-controller-manager -n kept
 
 The Lifecycle Toolkit and its dependencies are now installed and ready to use.
 
+**Note:** Installation of the Lifecycle Toolkit version 0.5.0 and lower is not supported via helm charts.
```

Comment:
```suggestion
**Note:** Versions 0.5.0 and lower of the Lifecycle Toolkit can only be installed using the manifest.  They cannot be installed from a Helm charts.
```

Reply:
For the `**note**` part, please look at the suggestion of this PR: https://github.com/keptn/lifecycle-toolkit/pull/962#pullrequestreview-1328233789